docs/solution-design.md

```diff
@@ -6,6 +6,50 @@ While there are many ways to achieve the goal we are looking for, for the purpos
 
 ## Components used and their roles
 
-## Message Flow
+The following components will be used as part of the solution:
+  
+- Cosmos DB Account
+- Cosmos DB Containers
+  - On Hand (onhand)
+  - Shipments (shipments)
+  - Master Data Container (mdc)
+- Azure Functions
+  - Change Feed Listeners
+  - Durable Function Orchestrator
+  - Durable Entity as a Virtual Actor
```

Comment:
```suggestion
  - Durable Entity (similar to a Virtual Actor)
```

Reply:
Might want to clarify the durable entity is at the store level (i.e. there will be numStores durable entities)

Reply:
Ok, will do that.